website/blog/2024-04-04-angular-upgrade-top-5-tips.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Angular Upgrade Journey: Top 5 things to keep in mind"
```

Comment:
Moving to which Angular version? Please mention it in the title. For example, "Ready to upgrade to Angular 16? Here's what you need to know"